packages/fxa-settings/src/models/Account.ts

```diff
@@ -559,17 +559,24 @@ export class Account implements AccountData {
     service?: string,
     redirectTo?: string
   ): Promise<PasswordForgotSendCodePayload> {
-    let serviceName;
+    let options: {
+      service?: string;
+      resume?: string;
+      redirectTo?: string;
+    } = {
+      resume: 'e30=', // base64 json for {}
+    };
+
+    // Important! Only set service when it's Firefox Sync
     if (service && service === MozServices.FirefoxSync) {
-      serviceName = 'sync';
-    } else {
-      serviceName = service;
+      options.service = 'sync';
     }
-    const result = await this.authClient.passwordForgotSendCode(email, {
-      service: serviceName,
-      resume: 'e30=', // base64 json for {}
-      redirectTo,
-    });
+
+    if (redirectTo) {
+      options.redirectTo = redirectTo;
```

Comment:
Yep, that understanding seems correct to me. 

Concerning @vpomerleau's comment. I looked into this, and I think someone else should dig into it more before filing a follow up. It seems like it might be a non issue that is unrelated to this PR. 

As mentioned in another comment, this PR does not remove the ability to send a redirectTo link, rather it simply stops sending the `redirectTo` link for oauth integrations in a reset password flow. I believe, the scenario that Valerie describes is not applicable to this PR since it describes a scenario where we'd be using web integration logic and not oauth integration logic.